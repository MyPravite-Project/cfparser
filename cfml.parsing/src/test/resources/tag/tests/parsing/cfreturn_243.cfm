<cfcomponent>
  <cffunction name="foo" access="public" output="false" returntype="any">
    <cfreturn  >
  </cffunction>
</cfcomponent>