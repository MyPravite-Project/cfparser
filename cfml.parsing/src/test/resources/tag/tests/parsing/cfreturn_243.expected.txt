START:cfcomponent
  =========TAG=================
  <cfcomponent>
  <cffunction name="foo" access="public" output="false" returntype="any">
    <cfreturn  >
  </cffunction>
</cfcomponent>
  START:cffunction
    =========TAG=================
    <cffunction name="foo" access="public" output="false" returntype="any">
    <cfreturn  >
  </cffunction>
    START:cfreturn
      =========TAG=================
      <cfreturn  >
    END:cfreturn
  END:cffunction
END:cfcomponent
